Microsoft.Toolkit.Uwp.Services/Services/Bing/BingDataProvider.cs

```diff
@@ -41,7 +42,7 @@ public class BingDataProvider : DataProviderBase<BingSearchConfig, BingResult>
         protected override async Task<IEnumerable<TSchema>> GetDataAsync<TSchema>(BingSearchConfig config, int maxRecords, IParser<TSchema> parser)
         {
             var countryValue = config.Country.GetStringValue();
-            var locParameter = string.IsNullOrEmpty(countryValue) ? string.Empty : $"loc:{countryValue}+";
+            var locParameter = string.IsNullOrEmpty(countryValue) ? $"loc:{CultureInfo.CurrentCulture.Name.Split('-')[1].ToLower()}+" : $"loc:{countryValue}+";
```

Comment:
Is CultureInfo.CurrentCulture.TwoLetterISOLanguageName different from this value?